src/Symfony/Component/HttpFoundation/Exception/AbstractHostException.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\Exception;
+
+/**
+ * THe HTTP request contains incorrect host data.
+ *
+ * @author SpacePossum
+ */
+abstract class AbstractHostException extends \UnexpectedValueException implements ExceptionInterface
+{
+    /**
+     * @var string
```

Comment:
They are useless, they can be inferred from the constructor's doc and most tools support this.